pkg/minikube/cluster/commands.go

```diff
@@ -16,7 +16,15 @@ limitations under the License.
 
 package cluster
 
+import "fmt"
+
+const (
+	remoteLocalKubeErrPath = "/var/log/localkube.err"
```

Comment:
I don't really see the purpose of these constants if they aren't used where they should be used.
They are still hard-coded in startCommand.